.github/workflows/integration-tests-on-release.yaml

```diff
@@ -6,7 +6,6 @@ on:
     branches:
       - master
       - release-*
-
```

Comment:
Please revert this change, no need for it now

Reply:
This means to pur back the 'paths-ignore' or simply just remove the blank line? I will use this to ammend and push with a better commit message

Reply:
I mean to add back that blank line so no changes show for this file.